tests/function_libs/torch_lib/e2e_ops_tests.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import itertools
+import unittest
+
+import onnxruntime
+import torch
+
+from tests.common import testutils
+
+
+class TorchLibe2eTest(testutils.TestBase):
```

Comment:
We probably don't need to subclass test base

Reply:
I used this to understand the cases working or not working. I was hoping to remove it once it is done. But I still have this issue with unknown types. Either I try to understand why this is happening can working this the automated tests is difficult or I update the converter to handle the case when type is unknown.

Reply:
I see. Thanks!

Reply:
I kept one example to avoid redoing this again.